### PR TITLE
Feature/420/admin preview

### PR DESCRIPTION
**Pull request by antoineeudes**

### CODE REVIEW

File: src/components/templates/EntranceExperience/EntranceExperience.tsx [hunk 1]
```diff
@@ -56,6 +57,8 @@ export const EntranceExperience: React.FunctionComponent<EntranceExperienceProps
       Date.now() / 1000
   );
 
+  const isUserVenueOwner = user && venue?.owners?.includes(user.uid);
```
Comment (maxb94) [inline src/components/templates/EntranceExperience/EntranceExperience.tsx:60]: venue is defined

File: src/components/templates/EntranceExperience/EntranceExperience.tsx [hunk 2]
```diff
@@ -270,6 +273,21 @@ export const EntranceExperience: React.FunctionComponent<EntranceExperienceProps
                   })}
                 </>
               )}
+            {isUserVenueOwner && (
+              <InformationCard title="Enter the venue as an admin">
```
Comment (maxb94) [inline src/components/templates/EntranceExperience/EntranceExperience.tsx:277]: Not keen on this copy. Should be something like "Preview what an event looks like in your venue". Please confirm text with Mathieu Robert.

File: src/pages/VenuePage/VenuePage.tsx [hunk 3]
```diff
@@ -53,6 +53,8 @@ const VenuePage = () => {
     currentTimestamp >
       event.start_utc_seconds + event.duration_minutes * ONE_MINUTE_IN_SECONDS;
 
+  const isUserVenueOwner = user && venue?.owners?.includes(user.uid);
```
Comment (maxb94) [inline src/pages/VenuePage/VenuePage.tsx:56]: venue is defined

File: src/pages/VenuePage/VenuePage.tsx [hunk 4]
```diff
@@ -65,30 +67,32 @@ const VenuePage = () => {
...
-    );
+    if (!canUserJoinTheEvent(event)) {
+      return (
+        <CountDown
+          startUtcSeconds={event.start_utc_seconds}
+          textBeforeCountdown="Bar opens in"
+        />
+      );
+    }
   }
```
Comment (maxb94) [inline src/pages/VenuePage/VenuePage.tsx:97]: would be nice to include a div at the top of the page saying something like "This is a preview of your venue during an event" in big bold letters. What do you think?

Comment (antoineeudes) [reply src/pages/VenuePage/VenuePage.tsx:97]: good idea

**Pull request merged**